packages/js/src/classic-editor/initial-state/index.js

```diff
@@ -0,0 +1,58 @@
+import {
+	getContent,
+	getDate,
+	getExcerpt,
+	getFeaturedImage,
+	getFocusKeyphrase,
+	getMetaDescription,
+	getPermalink,
+	getSeoTitle,
+	getSlug,
+	getTitle,
+	isCornerstone,
+} from "../dom";
```

Comment:
Not sure about this file name. I used the same file name as Nolle for these 'getters'.